site/blog/2022-10-28-optimize-s3-endpoints.mdx

```diff
@@ -0,0 +1,126 @@
+---
+slug: optimize-your-s3-endpoints
+title: Optimize your S3 endpoints for cost and efficiency
+authors: [yrobla]
+---
+
+# Optimize your S3 endpoints for cost and efficiency
+
+Are you using S3 buckets as part of your cloud deployments? How are you accessing them?
+
+When not configured properly, access to the S3 buckets may need internet access. One simple but costly way
```

Comment:
"may need internet access" reads a bit confusingly, since technically all S3 access is going to be done with a TCP connection. Perhaps:

```suggestion
When not configured properly, access to the S3 buckets may need to be accessed over the public internet. One simple but costly way
```